test/web/10-api.js

```diff
@@ -698,6 +699,55 @@ describe('Profile Manager API', () => {
       error.name.should.equal('TypeError');
       error.message.should.contain('profileId');
     });
+    it('should fail if no edvId', async () => {
+      const {parentCapabilities, hmac, keyAgreementKey} = mockData;
+      delete parentCapabilities.edv;
+      let error, result = null;
```

Comment:
style does not allow for multiple variables defined on a single line.  Applies throughout.

Reply:
@mandyvenables looks like this still has not been addressed.

Reply:
@mattcollier it has been updated to `const {
        parentCapabilities,
        edvId,
        hmac,
        keyAgreementKey
      } = mockData;`

Is that incorrect?

Reply:
I'm referring to the line I'm commenting on ` let error, result = null;`

our style does not allow defining multiple variable on a single line.

Reply:
Got it, updated it in the previous tests as well.